src/features/game/types/game.ts

```diff
@@ -409,6 +409,17 @@ export type HayseedHank = {
   };
 };
 
+export type Mushroom = {
+  amount: number;
```

Comment:
consider including `name: "Mushroom"` as well. This can future proof us when we add `Magic Mushroom`